.factorypath

```diff
@@ -0,0 +1,163 @@
+<factorypath>
```

Comment:
I'm not accustomed to seeing [Eclipse specific configuration files](https://stackoverflow.com/questions/42949198/automatically-generate-factorypath-on-project-import-when-using-maven-project-i) added to the Jenkins repositories.  I may be mistaken on that, but I don't recall that being common.

Unless others guide differently, I would not include files that are specific to Eclipse IDE in the pull request.